massgov/mayflower
# React / resolve console error for header searchRedirect proptype

- [ ] fix https://jira.mass.gov/browse/DP-16673
- [ ] fix https://jira.mass.gov/browse/DP-16676
- [ ] fix button search aria label attribute in header story
<img width="899" alt="Screen Shot 2019-12-06 at 12 56 34 PM" src="https://user-images.githubusercontent.com/5789411/70346677-d9cdeb00-182c-11ea-95ed-b863b4271fdd.png">


To test:
- check out this branch and run `npm start` in react
- go to http://localhost:6006/?path=/story/organisms-header--header-with-mainnav-utilitynav
- inspect and see the console errors above resolved
